src/index.html

```diff
@@ -29,7 +29,9 @@
             <div class="container">
                 <div class="row">
                     <div class="col-lg-12">
-                        <img class="cui-logo" src="img/informrus.svg" alt="Inform R US">
+                        <a href="/">
+                            <img class="cui-logo" src="img/informrus.svg" alt="Inform R US">
```

Comment:
I think this can be a standalone issue and does not have to be a requirement for merge, tbh. I'll make an issue